packages/nimble-components/src/table-column/number-text/models/tests/decimal-formatter.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { getSpecTypeByNamedList } from '../../../../utilities/tests/parameterized';
+import { DecimalFormatter } from '../decimal-formatter';
+
+describe('DecimalFormatter', () => {
+    const testCases: readonly {
+        name: string,
+        decimalDigits?: number,
```

Comment:
Perhaps there should be a case that tests the `undefined` case?

Reply:
There are a few test cases that don't specify a value for `decimalDigits` (-∞, ∞, and NaN), but you're right, there isn't a reason why those can't specify a value so that the test as a whole can be simplified.

Reply:
I updated the test to make `decimalDigits` required.